specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json

```diff
@@ -81,6 +80,107 @@
     }
   },
   "definitions": {
+    "AnalyzeConversationTaskKind":{
+      "type": "string",
+      "description": "Enumeration of supported Conversation tasks.",
+      "enum": [
+        "CustomConversation",
+        "PretrainedConversation"
+      ],
+      "x-ms-enum": {
+        "name": "AnalyzeConversationTaskKind",
+        "modelAsString": true
+      }
+    },
+    "AnalyzeConversationTaskResultsKind":{
+      "type": "string",
+      "description": "Enumeration of supported conversational task results",
+      "enum": [
+        "CustomConversationResult",
+        "PretrainedConversationResult"
+      ],
+      "x-ms-enum": {
+        "name": "AnalyzeConversationTaskResultsKind",
+        "modelAsString": true
+      }
+    },
+    "AnalyzeConversationTask": {
+      "discriminator": "kind",
+      "required": [
+        "kind"
+      ],
+      "properties": {
+        "kind": {
+            "$ref": "#/definitions/AnalyzeConversationTaskKind"
+          }
+      }
+    },
+    "AnalyzeConversationTaskResult": {
+      "discriminator": "kind",
+      "required": [
+        "kind"
+      ],
+      "properties": {
+        "kind": {
+          "$ref": "#/definitions/AnalyzeConversationTaskResultsKind"
+        }
+      }
+    },
+    "CustomConversationalTask":{
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeConversationTask"
+        }
+      ],
+      "properties": {
+        "analysisInput": {
+          "$ref": "#/definitions/ConversationAnalysisOptions"
+        },
+        "parameters": {
+          "$ref": "#/definitions/CustomConversationTaskParameters"
+        }
+      },
+      "x-ms-discriminator-value": "CustomConversation"
+    },
+    "CustomConversationTaskParameters":{
+      "type": "object",
+      "description": "Input parameters necessary for a Custom Conversation Task.",
+      "properties": {
+        "projectName":{
+          "type":"string",
+          "description": "The name of the project to use."
+        },
+        "deploymentName":{
```

Comment:
There are some body parameters for these, but maybe they don't work in this case. Please double check, though.